Fixed Invalid JSON

jaskirat8 opened the pull request:
Removed extra comma from variables.

azurecla commented:
Hi __@stefsch__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (stefsch). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

stefsch commented:
#sign-off

tysonn commented:
Please resubmit through the private repo, see email for details.

jaskirat8 committed:
Fixed Invalid JSON

Removed extra comma from variables.

jaskirat8 committed:
Added DNS as parameter

Introduced DNS as parameter. Will help kickstart template by copy pasting directly even in Portal UI and Cli.
